hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/ExternalTableIT.java

```diff
@@ -33,6 +35,14 @@
 
     @BeforeClass
     public static void cleanoutDirectory() {
+        try {
+            File file = new File(getExternalResourceDirectory());
+            if (file.exists())
+                FileUtils.deleteDirectory(new File(getExternalResourceDirectory()));
+            file.mkdir();
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
     }
```

Comment:
why are checking for the existence of the file after you catch the exception ? why not checking before you try to read or create a empty parquet file ? Is this for performance reason ?